src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
+  const repeatedValueFinder = [];
+
+  for (const key in robot) {
+    repairedRobot[robot[key]] = key;
+
+    if (repeatedValueFinder.includes(robot[key])) {
+      return null;
+    }
+
+    repeatedValueFinder.push(robot[key]);
+  }
```

Comment:
why not simply check if that key already exists in the object? you don't need the array here